src/born_digital_docs_scripts/make_sc.py

```diff
@@ -0,0 +1,147 @@
+from pathlib import Path
+import logging
+import subprocess 
+import argparse
+import re
+
+# Accept a directory (could hardcode or argparse)
+def parse_args() -> argparse.Namespace:
+    def extant_path(p: str) -> Path:
+        path = Path(p) 
+        if not path.exists():
+            raise argparse.ArgumentTypeError(f"{path} does not exist")
+        return path
+    
+    # def rclone_remote(p: str) -> Path:
+    #     if not re.match(r'*:*', p):
```

Comment:
the `*` (zero or more of) should be replaced by `.+`, (any character, 1 or more of them)